backend/utils/deployShop.js

```diff
@@ -42,6 +42,36 @@ function urlToMultiaddr(v, opts) {
   )}${url.pathname}`
 }
 
+async function configureShopDNS({
```

Comment:
I'll add some warning logging but don't want to add any thrown errors as that would change the behavior.  Especially since this is just moving existing code into a contained function.